gobblin-data-management/src/main/java/gobblin/data/management/copy/replication/EndPointCopyPair.java

```diff
@@ -0,0 +1,12 @@
+package gobblin.data.management.copy.replication;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@AllArgsConstructor
+@Data
+public class EndPointCopyPair {
```

Comment:
nit: Can we use CopyRoute instead of CopyPair? May be we can have routes with intermediate hops in the future? It's better to have the names describe the concept (the route to copy data from point A to point B) rather than how it is implemented (a pair of from- and to- points).